### DISCUSSION TITLE
Cola integration

### DISCUSSION THREAD

**Discussion by adam-hartshorne** [thomaspinder/GPJax#369] [Ideas]
It appears like there is a compatibility issue with plum-dispatch between GPJax and Cola. Cola wants to a cola-plum-dispatch v0.1.1 installed and GPJax install plum-dispatch v2.1.0. With both of these installs, Cola appears to unhappy and won't work. I have had to uninstall plum-dispatch v2.1.0 in order to get both GPJax and Cola to work together.

**Comment by thomaspinder** [comment]
I'm transferring this issue to a discussion @adam-hartshorne and dropping the bug tag. We don't publicly support cola and are working hard to establish an integration. This discussion can be used to raise points and ideas around how to best establish an integration.
